Changelog.md

```diff
@@ -5,6 +5,11 @@ Language Features:
 
 Compiler Features:
 * ethdebug: Experimental support for instructions and source locations under EOF.
+* DocString Parser: Deprecation of inline assembly special comment `memory-safe-assembly`.
+* Syntax Checker: Deprecation of ABI coder v1.
+* Syntax Checker: Deprecation of `virtual` modifiers.
+* Type Checker: Deprecation of `send` and `transfer` functions on instances of `address`.
+* Type Checker: Deprecation of comparisons between `contract`-typed variables.
```

Comment:
```suggestion
* DocString Parser: Warn about deprecation of inline assembly special comment `memory-safe-assembly`.
* Syntax Checker: Warn about deprecation of ABI coder v1.
* Syntax Checker: Warn about deprecation of virtual modifiers.
* Type Checker: Warn about deprecation of `send` and `transfer` functions on instances of `address`.
* Type Checker: Warn about deprecation of comparisons between variables of contract types.
```

Reply:
out of curiosity, if this pr isn't it, when was (or will be) the actual deprecation event? :)